providers/standard/src/airflow/providers/standard/triggers/external_task.py

```diff
@@ -143,11 +141,11 @@ def _get_count(self, states: typing.Iterable[str] | None) -> int:
 
 class DagStateTrigger(BaseTrigger):
     """
-    Waits asynchronously for a DAG to complete for a specific logical date.
+    Waits asynchronously for a DAG to complete for a specific run_id.
```

Comment:
```suggestion
    Waits asynchronously for a dag to complete for a specific run_id.
```
according to the discussion in https://lists.apache.org/thread/lktrzqkzrpvc1cyctxz7zxfmc0fwtq2j, I think we should use dag instead